src/core/render/index.js

```diff
@@ -444,7 +445,7 @@ export function initRender(vm) {
   }
```

Comment:
- [ ] I think we should avoid adding classes to a custom navbar here.

Reply:
I don't understand what this comment is referring to.